packages/apollo/src/index.ts

```diff
@@ -7,10 +7,10 @@ import {
   RequestHandler,
   split,
   WatchQueryFetchPolicy
-} from '@apollo/client/core'
-import { setContext } from '@apollo/client/link/context'
-import { GraphQLWsLink } from '@apollo/client/link/subscriptions'
-import { getMainDefinition } from '@apollo/client/utilities'
+} from '@apollo/client/core/index.js'
```

Comment:
why do we have now to give the full path of the js file?
If this is mandatory, is there a way to raise an error when building the package? For the moment, both vs code / eslint nor `pnpm build` accept a value like `@apollo/client/core`